style-dictionary/visual-refresh/metadata/borders.ts

```diff
@@ -109,6 +109,16 @@ const metadata: StyleDictionary.MetadataIndex = {
     public: true,
     themeable: true,
   },
+  borderWidthAlert: {
+    description: 'The border width of alerts.',
+    public: true,
+    themeable: true,
+  },
+  borderWidthField: {
+    description: 'The border width of fields.',
```

Comment:
other design tokens typically refer to "form fields" rather than just "fields"

Reply:
Updated!